addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -61,9 +61,9 @@ var MassMailingFieldHtml = FieldHtml.extend({
 
         var $editable = this.wysiwyg.getEditable();
         await this.wysiwyg.cleanForSave();
-        return this.wysiwyg.saveModifiedImages(this.$content).then(function () {
+        return this.wysiwyg.saveModifiedImages(this.$content).then(async function () {
```

Comment:
while you are at it, can you use arrow function ?
```suggestion
        return this.wysiwyg.saveModifiedImages(this.$content).then(async () => {
```